extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -192,7 +195,7 @@ describes.realWin('amp-bind', {
   });
 
   it('should verify string attribute bindings in dev mode', () => {
-    env.sandbox.stub(window, 'AMP_MODE', {development: true});
+    env.sandbox.stub(getMode(), 'development', true);
```

Comment:
Alright, fine. 😄  Done.

BTW, [property stubbing works in sandbox](https://github.com/sinonjs/sinon/issues/161#issuecomment-7014983) and is nice since it restores the original property.